cluster/gce/cni.template

```diff
@@ -7,11 +7,10 @@
       "mtu": 1460,
       "ipam": {
         "type": "host-local",
-        "subnet": "{{.PodCIDR}}",
+        "ranges": [{{range $i, $range := .PodCIDRRanges}}{{if $i}}, {{end}}[{"subnet": "{{$range}}"}]{{end}}],
         "routes": [
-          {
-            "dst": "0.0.0.0/0"
-          }
+          {"dst": "0.0.0.0/0"},
```

Comment:
we had issues in kind using a config like this on hosts without IPv6 enabled, so we had to check if ipv6 was enabled to add the IPv6 default route, don't know if this can happen here

Reply:
Good catch. Let me see.

Reply:
Kubenet always set it https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go#L286

Reply:
Based on my experiment, it seems fine:
```console
$ sysctl net/ipv6/conf/all/disable_ipv6
net.ipv6.conf.all.disable_ipv6 = 1
$ cat /etc/cni/net.d/10-containerd-net.conflist
{
  "cniVersion": "0.3.1",
  "name": "containerd-net",
  "plugins": [
    {
      "type": "bridge",
      "bridge": "cni0",
      "isGateway": true,
      "ipMasq": true,
      "promiscMode": true,
      "ipam": {
        "type": "host-local",
        "ranges": [
          [{
            "subnet": "10.88.0.0/16"
          }]
        ],
        "routes": [
          { "dst": "0.0.0.0/0" },
          { "dst": "::/0" }
        ]
      }
    },
    {
      "type": "portmap",
      "capabilities": {"portMappings": true}
    }
  ]
}
$ crictl runp sandbox.json
0103ae22345ebed28501b0388360b2f25cf0c112736f24c96883efd46733fbe1
```

Reply:
this was the issue I was referring https://github.com/kubernetes-sigs/kind/issues/626, seems that only happen if ipv6 is disabled at boot time

Reply:
If that is the case, it is a bug in kubenet in Kubernetes 1.16 that we need to fix.

Let me look into it tomorrow.

Reply:
Thanks for the good catch! Filed https://github.com/kubernetes/kubernetes/pull/82800 for kubenet, and will fix this PR. @aojea

Reply:
Done